docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,253 @@
 
-Working with benefits through the Finch API is a simple two-phase process:
-1. Benefits are configured at the company level. When a new benefit is created or an existing benefit is read from the system, you will receive a `benefit_id` for it.
-2. You can use the `benefit_id` to manage the enrollment of employees in that benefit.
+## How do payroll benefit deductions work?
 
-----
+Since benefits work very differently across different providers, it is important to know a little about how payroll and benefits work in general.
+
+Each payroll contains four important dates to know.
+
+1. Payroll Start Date - The first day of the pay period
+2. Payroll End Date - The last day of the pay period
+3. Payroll Close Date - The last date to make changes for that pay period
+4. Paycheck Date - The date on which employees are paid
+
+When considering benefit deductions, it is important to submit any employee deductions before the pay close date in order to affect for the current pay period. If any deductions are submitted after the payroll close date, they will only affect the next pay period, not the current. Since each payroll close date is different per provider, it is important to know this date and set proper expectations with your customers.
+
+Finch has two types of ways to call our API: Automated and Assisted.
+
+- If the payroll provider is ***automated***, any deductions that are submitted via our API will be changed in the provider’s system instantaneously.
+- If the payroll provider is ***assisted***, any deductions that are submitted via our API will be changed within 7-14 days. This is because a real Finch employee will read your API request and contact the provider directly, manually assist in creating a benefit and enrolling employees in that benefit, and send a response back to you when the request is complete. This benefits you because your team does not need to build a product operations team to make these changes yourself; Finch does this for you.
+
+You can review [our list of valid providers](https://developer.tryfinch.com/docs/reference/96f5be9e0ec1a-providers) to figure out which providers are automated vs assisted for the Benefits endpoints.
+
+## Benefits submissions
+
+Because of the manual nature of assisted benefit connections, it is important to submit payroll benefit deductions to our API 7 days before the customer’s payroll close date. This gives the Finch team a higher chance of making the change in the provider’s system so it can be processed with the current payroll period (unless explicitly specified otherwise in your API request).
+
+For most of your customers, this is already a part of the HR admin workflow. We are just changing the location of where they make this change. It is important as the developer to make it clear to your customers what the payroll close date is so that it can be communicated properly if the benefit will take effect this pay period or next pay period.
```

Comment:
I'm not sure I understand this section; what location are we changing for the HR admin?

Reply:
Instead of the HR admin making the change directly in the payroll system, the change will happen in the developers application which will initiate the change through Finch. I can remove it if it is confusing.

Reply:
Gotcha, makes sense. So the developer is now initiating the change via API, but the HR admin is providing the details e.g. around close date? (I don't know much about how close dates work. Do they vary per employer or per provider?) Maybe we can rephrase as "For most employers, this information is already needed for the HR admin workflow. As the developer, it is important to understand what the payroll close date is so that it can be communicated properly if the benefit will take effect this pay period or next pay period."

Reply:
This is excellent! Exactly what I was trying to say, just better 😜